docs/src/refindex.md

```diff
@@ -0,0 +1,4 @@
+# Index
```

Comment:
I think we agree that the API reference is useful. If one doesn't know the exact name of something, it may be easier to find in an index. I don't have a strong opinion on keeping it and we can of course also remove the index.

Reply:
The API reference is absolutely necessary, I agree. I personally have never needed an index, but of course that is very subjective. I have no qualms about keeping the index, I just stumbled across the names "reference" and "index" and just didn't know what is what until I opened them. But feel free to keep it as it is or remove the index, both is fine 👍

Reply:
Removed in 06b88b6